docs/specifications/broker-authentication.md

```diff
@@ -0,0 +1,107 @@
+# Broker Authentication
+
+This document attempts to outline the high-level plan for adding authentication
+to the broker api routes.
+
+When talking to service brokers, the [Service Catalog uses basic auth](#service-catalog-basic-auth). The Ansible Service Broker will implement Basic Auth using a secret shared between the catalog and the broker. The authentication will be configurable allowing a user i.e. developer to disable it.
+
+The plan for this sprint is to do the following:
+
+* make auth routes configurable by enhancing the configuration
+```
+auth:
+   - type: basic | INSERT OTHER TYPE HERE
+   - enable: true | false
```

Comment:
I don't have a strong opinion. While I think that auth is only a configuration for the broker. What about adding a key for specfic web facing "handler" configuration? It seems to be that the broker configuration is getting overloaded.

example:
```yaml
broker:                       
  launch_apb_on_bind: false
  recovery: true           
  output_request: true   
handler:
  dev_broker: true 
  output_request: true
  auth:                      
    type: basic              
    enable: true  
````